src/compiler/checker.ts

```diff
@@ -21447,7 +21447,8 @@ namespace ts {
 
         function checkJSDocParameterTag(node: JSDocParameterTag) {
             checkSourceElement(node.typeExpression);
-            if (!getParameterSymbolFromJSDoc(node)) {
+            const decl = getHostSignatureFromJSDoc(node);

```

Comment:
`getHostSignatureFromJSDoc` looks expensive and we won't use it if the param has a symbol; better to put it in an inner `if`.

Reply:
Yeah, good idea. It's also called twice -- once in getParameterSymbolFromJSDoc -- but I don't know that it's worthwhile to fix that.